tests/spread/charms/k8s-operator/task.yaml

```diff
@@ -0,0 +1,29 @@
+summary: pack external k8s operator charms
+manual: true
+systems:
+  - ubuntu-22.04-64
+kill-timeout: 30m
+
+environment:
+  CHARM/alertmanager: https://github.com/canonical/alertmanager-k8s-operator
+  CHARM/grafana: https://github.com/canonical/grafana-k8s-operator
+  CHARM/mysql: https://github.com/canonical/mysql-k8s-operator
+  CHARM/postgresql: https://github.com/canonical/postgresql-k8s-operator
+  CHARM/prometheus: https://github.com/canonical/prometheus-k8s-operator
+  CHARM/traefik: https://github.com/canonical/traefik-k8s-operator
+  CHARM/mattermost: https://git.launchpad.net/charm-k8s-mattermost
+  CHARM/discourse: https://git.launchpad.net/charm-k8s-discourse
+
+prepare: |
+  git clone --depth=1 "${CHARM}" charm
+
+restore: |
```

Comment:
could all these `restore`s eventually become a single `restore-each`? They are all very similar

Reply:
there's more to into this suite that might not fit, let's finish the work and find the common grounds